### PR TITLE
Lesson8_Task1

### PR DESCRIPTION
**Pull request by stalgatuly**
1) Write two services for a web application and corresponding docker files.
2) Write a docker-compose file for these two docker containers

### CODE REVIEW

File: lesson_08/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: "1"
```
Comment (nikonok) [inline lesson_08/docker-compose.yaml]: Are you using so old docker-compose version? My current version is 3.3

File: lesson_08/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+services:
+    saver:    
+        build: .\saver\
```
Comment (nikonok) [inline lesson_08/docker-compose.yaml]: That path is not valid for me, @denislituev is that the same for you?
I propose to use `saver/`

File: lesson_08/docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        # image: web_epam
+        ports:
+            - "8080:8080"
```
Comment (nikonok) [inline lesson_08/docker-compose.yaml:7]: Using of two equal ports in two different services

File: lesson_08/handler/handler.go [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+			MaxAge: 300,
+		}
+		// setting the cookie
+		http.SetCookie(w, cookie)
+		newToken := saver.Token{
+			Token:     cookie.Value,
+			CreatedAt: time.Now().Local().String(),
+			ExpiredAt: time.Now().Local().Add(time.Hour * time.Duration(240)).String(),
+		}
+		saver.Tokens[int64(len(saver.Tokens))] = newToken
```
Comment (nikonok) [inline lesson_08/handler/handler.go:74]: You need to send token over network. So the second service should open the port, listen to connections and so on.

**Review by nikonok — CHANGES_REQUESTED** [review]

File: lesson_08/docker-compose.yaml (hunk 2 above)
Comment (denislituev) [reply lesson_08/docker-compose.yaml]: I think it should be` saver/`

**Review by denislituev — CHANGES_REQUESTED** [review]

**Pull request merged**